.github/workflows/ci.yml

```
@@ -30,6 +30,7 @@ jobs:
         uses: golangci/golangci-lint-action@v7
         with:
           args: --timeout=5m
+          verify-config: false
```

actionlint (1.7.12)

[error] input "verify-config" is not defined in action "golangci/golangci-lint-action@v7". available inputs are "args", "cache-invalidation-interval", "github-token", "install-mode", "only-new-issues", "problem-matchers", "skip-cache", "skip-save-cache", "verify", "version", "working-directory"

(action)